controllers/topics.js

```diff
@@ -0,0 +1,14 @@
+const {findTopics} = require('../models/topics')
+
+function getTopics (req,res,next) {
+  return findTopics().then(({rows})=>{
+    const topics = rows
+    res.status(200).send(topics)
```

Comment:
anything your controllers send back need to be sent back on a meaningful key ie. .send({topics})

Reply:
Would that still be the case even though the task specifies that it should return an array?